Circle_Sort/circle_sort.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+//Circle sort algorithm can be visualized by drawing concentric circles on an array of integers. 
+//The elements of the array lying on the same circle diametrically opposite to each other are compared and if found in the wrong order they are swapped.
+// This goes on in a recursive fashion in which the array is divided into sub-arrays on which the above process is repeated until we get pairs of sorted elements which when put together form a sorted array.
+
+//function to perform circular swaps recursively.
+//this function returns true if there was a swap.
+//operation performed.
+
+function circleSortRec($a, $low, $high)
+{
+    $swapped = false;
+    
+    //base case
+
+    if($low == $high)
+    {
+        return false;
+    }
+
+    //storing upper and lower bounds of list to be used in the recursive case.
+
+    $lo = $low;
+    $hi = $high;
+
+    while($lo < $hi)
+    {
+        //swaps the pair of elements if True.
+
```

Comment:
Remove this line

Reply:
@somya-kapoor  done the required changes.